dev/Prototypes/ItemsViewPrototype/FileExplorerScenario.xaml

```diff
@@ -72,7 +78,10 @@
                 <c:RecyclingElementFactory RecyclePool="{StaticResource recyclePool}">
                     <DataTemplate x:Key="IconItemTemplate" x:DataType="dep:Item">
                         <l:ItemContainer Layout="{StaticResource iconItemLayout}" Background="LightGray" BorderBrush="Gray" BorderThickness="1">
-                            <Image Grid.Column="0" Width="60" Height="60">
+                            <l:DragAndDropBehavior.Source>
+                                <l:DragAndDropPayload Data="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}}" Operation="Copy" />
```

Comment:
Thinking about this, it might be better to pass the selection model as the payload.